pkg/cmd/drtprod/configs/drt_scale.yaml

```diff
@@ -21,7 +23,7 @@ targets:
           gce-managed: true
           gce-enable-multiple-stores: true
           gce-zones: "us-central1-a"
-          nodes: 150
+          nodes: $CLUSTER_NODES
           gce-machine-type: n2-standard-16
           local-ssd: true
           gce-local-ssd-count: 4
```

Comment:
It may be beneficial to make this a env variable as well as this is used in 2 places.

Reply:
Deferring this for now as per our offline discussion